tests/test_checker.py

```diff
@@ -226,3 +232,25 @@ def test_sbomchecker_output_html():
     )
 
     assert got == expected
+
+def test_components_without_functions():
+    logger = logging.getLogger(__name__)
+    logger.info("In test")
```

Comment:
Same logging comment as above.

Reply:
Removed in [dc3c53d](https://github.com/spdx/ntia-conformance-checker/pull/169/commits/dc3c53df654770de2a6b4015260b7bc2e2ea43ec)